notebook/automl_lightgbm.ipynb

```diff
@@ -39,7 +39,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%pip install flaml[notebook]==1.0.8"
```

Comment:
version 1.0.8 cause errors in this notebook?

Reply:
yes, the error is
AttributeError                            Traceback (most recent call last)
[<ipython-input-8-305c9cf35059>](https://localhost:8080/#) in <module>
      1 import matplotlib.pyplot as plt
----> 2 plt.barh(automl.feature_names_in_, automl.feature_importances_)

AttributeError: 'AutoML' object has no attribute 'feature_names_in_'